tests/all-tests.js

```diff
@@ -55,7 +55,9 @@ var testIncludes = [
     "../gpii/node_modules/preferencesServer/test/preferencesServerTests.js",
     "../gpii/node_modules/rawPreferencesServer/test/RawPreferencesTest.js",
     "../gpii/node_modules/ontologyHandler/test/node/OntologyHandlerTests.js",
-    "../gpii/node_modules/contextManager/test/ContextManagerTests.js"
+    "../gpii/node_modules/contextManager/test/ContextManagerTests.js",
+    // expects an internet connection and preferences server running at http://preferences.gpii.net
+    "./ProductionConfigTests.js"
```

Comment:
We can't have this in all-tests since it violates the assumptions of the rest of the testing file. You could create a new top-level driver named all-production-tests.js but since you just have the one production driver, we can just say that ProductionConfigTests.js is this driver, in which case you should add proper commenting at the head of the file to explain its assumptions and then add information to a new section in the "Testing" section of the overall universal README.md to explain this new file and its assumptions.